src/components/genes/Genes.js

```diff
@@ -6,6 +6,7 @@ export default function Genes(props) {
   const {
```

Comment:
Shouldn't we update the name of this component? Or are we avoiding breaking changes here?  Could export a deprecated/duplicated version...